src/Microsoft.ML.AutoML/EstimatorExtensions/EstimatorExtensions.cs

```diff
@@ -223,28 +227,94 @@ private static IEstimator<ITransformer> CreateInstance(MLContext context, string
         }
     }
 
-    internal class ImageFeaturizingExtension : IEstimatorExtension
+    internal class ImageLoadingExtension : IEstimatorExtension
     {
+        public static string ImageFolder { private get; set; }
         public IEstimator<ITransformer> CreateInstance(MLContext context, PipelineNode pipelineNode)
         {
             return CreateInstance(context, pipelineNode.InColumns[0], pipelineNode.OutColumns[0]);
         }
 
         public static SuggestedTransform CreateSuggestedTransform(MLContext context, string inColumn, string outColumn)
         {
-            var pipelineNode = new PipelineNode(EstimatorName.ImageFeaturizing.ToString(),
-                PipelineNodeType.Transform, inColumn, outColumn);
+            var pipelineNodeProperty = new Dictionary<string, object>()
+            {
+                { "imageFolder", ImageFolder },
+            };
+            var pipelineNode = new PipelineNode(EstimatorName.ImageLoading.ToString(), PipelineNodeType.Transform, inColumn, outColumn, pipelineNodeProperty);
             var estimator = CreateInstance(context, inColumn, outColumn);
             return new SuggestedTransform(pipelineNode, estimator);
         }
 
         private static IEstimator<ITransformer> CreateInstance(MLContext context, string inColumn, string outColumn)
         {
-            var estimator = context.Transforms.LoadImages(outputColumnName: outColumn, imageFolder: "/Users/justinormont/Documents/src/open-datasets/Datasets/DogBreedsVsFruits/Dataset", inputColumnName: inColumn)
-                .Append(context.Transforms.ResizeImages(outputColumnName: outColumn, imageWidth: 224, imageHeight: 224, inputColumnName: outColumn))
-                .Append(context.Transforms.ExtractPixels(outputColumnName: outColumn, interleavePixelColors: false, offsetImage: 0))
-                .Append(context.Transforms.DnnFeaturizeImage(outColumn, m => m.ModelSelector.ResNet18(context, m.OutputColumn, m.InputColumn), outColumn));
-            return estimator;
+            return context.Transforms.LoadImages(outColumn, ImageFolder, inColumn);
+        }
+    }
+
+    internal class ImageResizingExtension : IEstimatorExtension
+    {
+        public IEstimator<ITransformer> CreateInstance(MLContext context, PipelineNode pipelineNode)
+        {
+            return CreateInstance(context, pipelineNode.InColumns[0], pipelineNode.OutColumns[0]);
+        }
+
+        public static SuggestedTransform CreateSuggestedTransform(MLContext context, string inColumn, string outColumn)
+        {
+            var pipelineNodeProperty = new Dictionary<string, object>()
+            {
+                { "imageWidth", 224 },
+                { "imageHeight", 224 },
```

Comment:
The needed size will vary based on the model used.

Reply:
## resolved